src/web/modules/gateway_accounts/detail.njk

```diff
@@ -347,6 +348,47 @@
     </div>
   {% endif %}
 
+  {% if account.send_payer_email_to_gateway === true %}
+    <div>
+      <h1 class="govuk-heading-s">Sending payer email to gateway is enabled</h1>
+      <p class="govuk-body">If enabled, the paying user's email is sent to the payment gateway for fraud prevention purposes. This is not implemented for Stripe accounts.</p>
+
+      {{ govukWarningText({
+        text: "Ensure that the service does not have the setting enabled in Worldpay that will send emails to the payer.",
+        iconFallbackText: "Warning"
+      }) }}
+
+      <form method="POST" action="/gateway_accounts/{{ gatewayAccountId }}/toggle_send_payer_email_to_gateway">
+        {{ govukButton({
+          text: "Disable sending payer email to gateway"
+          })
+        }}
+        <input type="hidden" name="_csrf" value="{{ csrf }}">
+      </form>
+    </div>
+  {% endif %}
+  
+  {% if account.send_payer_email_to_gateway === false %}
+    <div>
+      <h1 class="govuk-heading-s">Enable sending payer email to gateway</h1>
+      <p class="govuk-body">If enabled, the paying user's email is sent to the payment gateway for fraud prevention purposes. This is not implemented for Stripe accounts.</p>
+      
+      {{ govukWarningText({
+        text: "Ensure that the service does not have the setting enabled in Worldpay that will send emails to the payer.",
+        iconFallbackText: "Warning"
+      }) }}
+
+      <form method="POST" action="/gateway_accounts/{{ gatewayAccountId }}/toggle_send_payer_email_to_gateway">
+        {{ govukButton({
+          text: "Enable sending payer email to gateway",
+          classes: "govuk-button--warning"
+          })
+        }}
+        <input type="hidden" name="_csrf" value="{{ csrf }}">
+      </form>
+    </div>
+  {% endif %}
```

Comment:
This is likely how it's done elsewhere but feels like duplication that could be avoided

I haven't checked the following syntax or run the code but an idea of how that could be accomplished:
```suggestion
  {% set sendPayerEmailHeading = "Sending payer email to gateway is enabled" if account.send_payer_email_to_gateway else "Enable sending payer email to gateway" %}
  {% set sendPayerEmailButtonText = "Disable sending payer email to gateway" if account.send_payer_email_to_gateway else "Enable sending payer email to gateway" %}
    <div>
      <h1 class="govuk-heading-s">{{ sendPayerEmailHeading }}</h1>
      <p class="govuk-body">If enabled, the paying user's email is sent to the payment gateway for fraud prevention purposes. This is not implemented for Stripe accounts.</p>

      {{ govukWarningText({
        text: "Ensure that the service does not have the setting enabled in Worldpay that will send emails to the payer.",
        iconFallbackText: "Warning"
      }) }}

      <form method="POST" action="/gateway_accounts/{{ gatewayAccountId }}/toggle_send_payer_email_to_gateway">
        {{ govukButton({
          text: sendPayerEmailButtonText
          })
        }}
        <input type="hidden" name="_csrf" value="{{ csrf }}">
      </form>
    </div>
```

Reply:
I have updated it to reduce the duplication.

Couldn't get the `gov uk button` macro to work with variables.
So just decided to use `if` statements for know.